libsigner/src/messages.rs

```diff
@@ -927,11 +998,70 @@ impl SignerMessage {
             },
             Self::BlockResponse(_) => BLOCK_SLOT_ID,
             Self::Transactions(_) => TRANSACTIONS_SLOT_ID,
+            Self::Nack(_) => NACK_SLOT_ID,
         };
         SIGNER_SLOTS_PER_USER * id + slot_id
     }
 }
 
+/// Signed network packets sent by signers along with their view
+/// of stacks tip consensus hash and block height for
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub struct PacketInfo {
```

Comment:
Why is the packet struct PacketInfo, while the coordinator struct is CoordinatorMetadata?  It seems on first blush that they could have the same suffix.

Reply:
I can rename it to PacketMetadata instead of PacketInfo. 
Initially I only had one CoordinatorInfo that included CoordinatorMetadata fields in this file but I ran into issues for serialization/deserialization of PublicKey based on the StacksMessageCodec requirements so I decided to break it apart since I didn't really need the publicKey/coordinator_id as part of messages.

Reply:
I'd prefer the same suffix, but it's not a huge deal.